integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -2307,6 +2307,14 @@ msgstr ""
 "Einige Angebote konnten nicht deaktiviert werden, da sie in mindestens einer "
 "Seite eingebettet sind."
 
+#: cms/forms/regions/region_form.py
+msgid ""
+"Please provide a Zammad URL and Access Token before enabling the Integreat "
```

Comment:
Am I getting the tone here correctly and this is just a suggestion and not a hard requirement? :thinking:

Reply:
Hm, fair. I'll make it a demand 😁

Reply:
:grin: